pkg/util/arrays.go

```diff
@@ -15,12 +15,26 @@
 package util
 
 import (
+	"bytes"
+	"crypto/md5"
 	"fmt"
 	"strings"
 )
 
 const keyPairSeparator = "="
 
+// GenerateMD5Hash will generate a MD5 hash from the given map
+func GenerateMD5Hash(source map[string]string) string {
```

Comment:
I would suggest moving the function to a separate file, e.g. string.go or hash.go